### DISCUSSION TITLE
zed CLI open in current project/window (Sublime Text CLI `--add` equivalent)

### DISCUSSION THREAD

**Discussion by mkdynamic** [zed-industries/zed#6599] [Feature Requests]
Is there a way to open a file in the currently open project window (as a Tab, essentially) when using the Zed CLI? [Sublime Text has an `--add` flag for this](https://www.sublimetext.com/docs/command_line.html#usage).

I think this would be a desirable feature if it is not currently possible.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
